keyboards/anne_pro/anne_pro.c

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2019 Michiel Visser
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "anne_pro.h"
+#include "anne_pro_lighting.h"
+#include "anne_pro_bluetooth.h"
+#include "ch.h"
+#include "hal.h"
+
```

Comment:
Exploratory testing:
```suggestion

void early_hardware_init_pre(void) {
    /* Disable all external interrupt vectors, they are left enabled by the bootloader */
    nvicDisableVector(EXTI0_IRQn);
    nvicDisableVector(EXTI1_IRQn);
    nvicDisableVector(EXTI2_IRQn);
    nvicDisableVector(EXTI3_IRQn);
    nvicDisableVector(EXTI4_IRQn);
    nvicDisableVector(EXTI9_5_IRQn);
    nvicDisableVector(EXTI15_10_IRQn);
}

```